### DISCUSSION TITLE
PnP Search results and filter webparts

### DISCUSSION THREAD

**Discussion by AnouckF** [microsoft-search/pnp-modern-search#4108] [Q&A]
In the search results webpart, I've configured it to display the Site_title in a column so that end users can easily identify which site the information is coming from. 
![Search_Results_Configuration](https://github.com/user-attachments/assets/970b6074-7e7d-495a-a362-b39c79b29382)

Additionally, I've set up RefinableString21 to map with both OWS_SiteName and ows_Site_x0020_Name. 
![Screenshot_17-11-2024_73847_ijsfabriekstrombeek-admin sharepoint com](https://github.com/user-attachments/assets/0903c516-1895-4ea3-8f05-6dcc76f61a35)

However, when I search for a document, I notice that many of the site titles are not properly mapped, as shown in the attached image. This inconsistency is causing confusion among the end users. 
<img width="805" alt="result" src="https://github.com/user-attachments/assets/c1934820-4eea-4877-bf61-74a043e18b19">

How we can ensure that our RefinableString is consistently getting the same values as the Site Title used in the search results? Is there something I might be missing in the configuration? Your assistance would be greatly appreciated. Thanks a lot for your help and looking forward to your guidance on this matter!

**Comment by kasperbolarsen** [comment]
The API returns a maximum of refiner value set by MS, as per https://github.com/microsoft-search/pnp-modern-search/issues/1097 it seems to be 1000 at the moment. 
Might you be exceeding this limit?
